### ISSUE TITLE
Images are no longer part of the feed

### ISSUE THREAD

**Issue by yutsuku** [trashhalo/reddit-rss#33]
Recently I noticed in my feeds that images are now missing, which is one of the main reasons I keep using this.
Only entries with videos seems to contain a thumbnail image.

- Is this a side-effect of parser being broken right now?
- Is this some reddit API-limit issue?
- Will the images be back in the feed in the future?

Example feed obtained from https://reddit.0qz.fun/r/Animemes/.json
```
<?xml version="1.0" encoding="UTF-8"?><rss version="2.0" xmlns:content="http://purl.org/rss/1.0/modules/content/">
  <channel>
    <title>reddit-rss /r/Animemes/.json</title>
    <link>https://github.com/trashhalo/reddit-rss</link>
    <description>Reddit RSS feed that links directly to the content</description>
    <managingEditor>s@0qz.fun (Stephen Solka)</managingEditor>
    <pubDate>Tue, 31 Aug 2021 16:05:01 +0000</pubDate>
    <lastBuildDate>Tue, 31 Aug 2021 16:05:01 +0000</lastBuildDate>
    <item>
      <title>Do i have enough karma to post here?</title>
      <link>https://i.redd.it/yb7p6diy0pk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf5mfp/do_i_have_enough_karma_to_post_here/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf5mfp</guid>
      <pubDate>Tue, 31 Aug 2021 13:14:58 +0000</pubDate>
    </item>
    <item>
      <title>I&#39;m not mad, I&#39;m disappointed.</title>
      <link>https://i.redd.it/lf1963lfqnk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1xzc/im_not_mad_im_disappointed/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf1xzc</guid>
      <pubDate>Tue, 31 Aug 2021 08:51:32 +0000</pubDate>
    </item>
    <item>
      <title>strongest genshin player</title>
      <link>https://i.redd.it/ylcg77eapok71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf4hni/strongest_genshin_player/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf4hni</guid>
      <pubDate>Tue, 31 Aug 2021 12:06:54 +0000</pubDate>
    </item>
    <item>
      <title>Good job crayon eaters, good job.</title>
      <link>https://v.redd.it/cf8l76lhbmk71</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/peyexb/good_job_crayon_eaters_good_job/">comments</a></p> <iframe src="https://v.redd.it/cf8l76lhbmk71/DASH_1080.mp4?source=fallback" width="597" height="1080"/> <img src="https://b.thumbs.redditmedia.com/7YwKb9P4jwwIELQi_uxtRlCX5yAe1qvUKY40K3D32gM.jpg" class="webfeedsFeaturedVisual"/>]]></content:encoded>
      <author>v.redd.it</author>
      <guid>peyexb</guid>
      <pubDate>Tue, 31 Aug 2021 04:06:43 +0000</pubDate>
    </item>
    <item>
      <title>Daily Erwin meme #1306: installed up the ass</title>
      <link>https://i.redd.it/sokyqehccnk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1429/daily_erwin_meme_1306_installed_up_the_ass/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf1429</guid>
      <pubDate>Tue, 31 Aug 2021 07:36:01 +0000</pubDate>
    </item>
    <item>
      <title>You guys should try it for fun</title>
      <link>https://i.redd.it/iyh5j6rcgok71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf3rut/you_guys_should_try_it_for_fun/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf3rut</guid>
      <pubDate>Tue, 31 Aug 2021 11:16:51 +0000</pubDate>
    </item>
    <item>
      <title>Shake your device slowly for a better effect</title>
      <link>https://i.redd.it/64dm6r65slk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pewpay/shake_your_device_slowly_for_a_better_effect/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pewpay</guid>
      <pubDate>Tue, 31 Aug 2021 02:17:33 +0000</pubDate>
    </item>
    <item>
      <title>holy shit its bts</title>
      <link>https://i.redd.it/eikrfa3qxnk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2f5n/holy_shit_its_bts/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf2f5n</guid>
      <pubDate>Tue, 31 Aug 2021 09:32:23 +0000</pubDate>
    </item>
    <item>
      <title>zero two farts</title>
      <link>https://i.redd.it/0moua2tn3ok71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2tx6/zero_two_farts/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf2tx6</guid>
      <pubDate>Tue, 31 Aug 2021 10:05:40 +0000</pubDate>
    </item>
    <item>
      <title>Need to make sure nothing is inappropriate</title>
      <link>https://i.redd.it/2jn97y639pk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf6e1v/need_to_make_sure_nothing_is_inappropriate/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf6e1v</guid>
      <pubDate>Tue, 31 Aug 2021 13:57:56 +0000</pubDate>
    </item>
    <item>
      <title>this cherry looks like a butt</title>
      <link>https://i.redd.it/yww2wgkdbok71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf3dtv/this_cherry_looks_like_a_butt/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf3dtv</guid>
      <pubDate>Tue, 31 Aug 2021 10:48:55 +0000</pubDate>
    </item>
    <item>
      <title>Some things just never change</title>
      <link>https://i.redd.it/f76mi3uzmmk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pezb3g/some_things_just_never_change/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pezb3g</guid>
      <pubDate>Tue, 31 Aug 2021 05:10:38 +0000</pubDate>
    </item>
    <item>
      <title>Wait.... is this how it works?</title>
      <link>https://i.redd.it/gp76waqiink71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1goz/wait_is_this_how_it_works/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf1goz</guid>
      <pubDate>Tue, 31 Aug 2021 08:07:13 +0000</pubDate>
    </item>
    <item>
      <title>poor choice of words</title>
      <link>https://i.redd.it/os8e8ots7pk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf69b0/poor_choice_of_words/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf69b0</guid>
      <pubDate>Tue, 31 Aug 2021 13:50:39 +0000</pubDate>
    </item>
    <item>
      <title>I see no difference</title>
      <link>https://i.redd.it/pyaq5exfglk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pevm29/i_see_no_difference/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pevm29</guid>
      <pubDate>Tue, 31 Aug 2021 01:11:57 +0000</pubDate>
    </item>
    <item>
      <title>Monomie</title>
      <link>https://v.redd.it/opha42os5pk71</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf62mj/monomie/">comments</a></p> <iframe src="https://v.redd.it/opha42os5pk71/DASH_1080.mp4?source=fallback" width="1080" height="1080"/> <img src="https://b.thumbs.redditmedia.com/of1xqhxOV27Of93vg3xKzMRYrivVqbqaLHJgIAT2JFE.jpg" class="webfeedsFeaturedVisual"/>]]></content:encoded>
      <author>v.redd.it</author>
      <guid>pf62mj</guid>
      <pubDate>Tue, 31 Aug 2021 13:40:04 +0000</pubDate>
    </item>
    <item>
      <title>Oh No, The Cursed Hairstyle</title>
      <link>https://i.redd.it/6g6vnwro1lk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/peuf2k/oh_no_the_cursed_hairstyle/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>peuf2k</guid>
      <pubDate>Tue, 31 Aug 2021 00:01:33 +0000</pubDate>
    </item>
    <item>
      <title>Ahem... Tommorow is September.</title>
      <link>https://i.redd.it/d17gvyqfmok71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf491n/ahem_tommorow_is_september/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf491n</guid>
      <pubDate>Tue, 31 Aug 2021 11:51:13 +0000</pubDate>
    </item>
    <item>
      <title>twitter</title>
      <link>https://i.redd.it/nnw3d4h0wlk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pex276/twitter/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pex276</guid>
      <pubDate>Tue, 31 Aug 2021 02:39:27 +0000</pubDate>
    </item>
    <item>
      <title>no way</title>
      <link>https://i.redd.it/r78geqc72pk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf5paa/no_way/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf5paa</guid>
      <pubDate>Tue, 31 Aug 2021 13:19:17 +0000</pubDate>
    </item>
    <item>
      <title>I don&#39;t know what to name it</title>
      <link>https://i.redd.it/18mnx13p9pk71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf6gep/i_dont_know_what_to_name_it/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf6gep</guid>
      <pubDate>Tue, 31 Aug 2021 14:01:17 +0000</pubDate>
    </item>
    <item>
      <title>Yup!! No embarrassing stuff after this one!</title>
      <link>https://i.redd.it/1s976n602mk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pexlth/yup_no_embarrassing_stuff_after_this_one/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pexlth</guid>
      <pubDate>Tue, 31 Aug 2021 03:12:57 +0000</pubDate>
    </item>
    <item>
      <title>Mosquito, thats what I see.</title>
      <link>https://i.redd.it/rlh764gk4ok71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2w7x/mosquito_thats_what_i_see/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pf2w7x</guid>
      <pubDate>Tue, 31 Aug 2021 10:10:51 +0000</pubDate>
    </item>
    <item>
      <title>i wonder if it would melt...</title>
      <link>https://i.redd.it/3d4r6l0zyik71.jpg</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pem1ck/i_wonder_if_it_would_melt/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pem1ck</guid>
      <pubDate>Mon, 30 Aug 2021 16:50:32 +0000</pubDate>
    </item>
    <item>
      <title>Since when did watching anime start making you cool</title>
      <link>https://i.redd.it/otz69vnm4jk71.png</link>
      <description></description>
      <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pemonj/since_when_did_watching_anime_start_making_you/">comments</a></p> ]]></content:encoded>
      <author>i.redd.it</author>
      <guid>pemonj</guid>
      <pubDate>Mon, 30 Aug 2021 17:22:59 +0000</pubDate>
    </item>
  </channel>
</rss>
```

**Comment by trashhalo** [comment]
I didn't ev n know it broke!! Thanks for telling me

On Tue, Aug 31, 2021, at 12:18 PM, yutsuku wrote:
> 
> 
> 
> Recently I noticed in my feeds that images are now missing, which is one of the main reasons I keep using this.
> Only entries with videos seems to contain a thumbnail image.
> 
>  * Is this a side-effect of parser being broken right now?
>  * Is this some reddit API-limit issue?
>  * Will the images be back in the feed in the future?
> Example feed obtained from https://reddit.0qz.fun/r/Animemes/.json
> 
> `<?xml version="1.0" encoding="UTF-8"?><rss version="2.0" xmlns:content="http://purl.org/rss/1.0/modules/content/">
>   <channel>
>     <title>reddit-rss /r/Animemes/.json</title>
>     <link>https://github.com/trashhalo/reddit-rss</link>
>     <description>Reddit RSS feed that links directly to the content</description>
>     ***@***.*** (Stephen Solka)</managingEditor>
>     <pubDate>Tue, 31 Aug 2021 16:05:01 +0000</pubDate>
>     <lastBuildDate>Tue, 31 Aug 2021 16:05:01 +0000</lastBuildDate>
>     <item>
>       <title>Do i have enough karma to post here?</title>
>       <link>https://i.redd.it/yb7p6diy0pk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf5mfp/do_i_have_enough_karma_to_post_here/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf5mfp</guid>
>       <pubDate>Tue, 31 Aug 2021 13:14:58 +0000</pubDate>
>     </item>
>     <item>
>       <title>I&#39;m not mad, I&#39;m disappointed.</title>
>       <link>https://i.redd.it/lf1963lfqnk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1xzc/im_not_mad_im_disappointed/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf1xzc</guid>
>       <pubDate>Tue, 31 Aug 2021 08:51:32 +0000</pubDate>
>     </item>
>     <item>
>       <title>strongest genshin player</title>
>       <link>https://i.redd.it/ylcg77eapok71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf4hni/strongest_genshin_player/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf4hni</guid>
>       <pubDate>Tue, 31 Aug 2021 12:06:54 +0000</pubDate>
>     </item>
>     <item>
>       <title>Good job crayon eaters, good job.</title>
>       <link>https://v.redd.it/cf8l76lhbmk71</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/peyexb/good_job_crayon_eaters_good_job/">comments</a></p> <iframe src="https://v.redd.it/cf8l76lhbmk71/DASH_1080.mp4?source=fallback" width="597" height="1080"/> <img src="https://b.thumbs.redditmedia.com/7YwKb9P4jwwIELQi_uxtRlCX5yAe1qvUKY40K3D32gM.jpg" class="webfeedsFeaturedVisual"/>]]></content:encoded>
>       <author>v.redd.it</author>
>       <guid>peyexb</guid>
>       <pubDate>Tue, 31 Aug 2021 04:06:43 +0000</pubDate>
>     </item>
>     <item>
>       <title>Daily Erwin meme #1306: installed up the ass</title>
>       <link>https://i.redd.it/sokyqehccnk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1429/daily_erwin_meme_1306_installed_up_the_ass/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf1429</guid>
>       <pubDate>Tue, 31 Aug 2021 07:36:01 +0000</pubDate>
>     </item>
>     <item>
>       <title>You guys should try it for fun</title>
>       <link>https://i.redd.it/iyh5j6rcgok71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf3rut/you_guys_should_try_it_for_fun/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf3rut</guid>
>       <pubDate>Tue, 31 Aug 2021 11:16:51 +0000</pubDate>
>     </item>
>     <item>
>       <title>Shake your device slowly for a better effect</title>
>       <link>https://i.redd.it/64dm6r65slk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pewpay/shake_your_device_slowly_for_a_better_effect/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pewpay</guid>
>       <pubDate>Tue, 31 Aug 2021 02:17:33 +0000</pubDate>
>     </item>
>     <item>
>       <title>holy shit its bts</title>
>       <link>https://i.redd.it/eikrfa3qxnk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2f5n/holy_shit_its_bts/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf2f5n</guid>
>       <pubDate>Tue, 31 Aug 2021 09:32:23 +0000</pubDate>
>     </item>
>     <item>
>       <title>zero two farts</title>
>       <link>https://i.redd.it/0moua2tn3ok71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2tx6/zero_two_farts/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf2tx6</guid>
>       <pubDate>Tue, 31 Aug 2021 10:05:40 +0000</pubDate>
>     </item>
>     <item>
>       <title>Need to make sure nothing is inappropriate</title>
>       <link>https://i.redd.it/2jn97y639pk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf6e1v/need_to_make_sure_nothing_is_inappropriate/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf6e1v</guid>
>       <pubDate>Tue, 31 Aug 2021 13:57:56 +0000</pubDate>
>     </item>
>     <item>
>       <title>this cherry looks like a butt</title>
>       <link>https://i.redd.it/yww2wgkdbok71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf3dtv/this_cherry_looks_like_a_butt/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf3dtv</guid>
>       <pubDate>Tue, 31 Aug 2021 10:48:55 +0000</pubDate>
>     </item>
>     <item>
>       <title>Some things just never change</title>
>       <link>https://i.redd.it/f76mi3uzmmk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pezb3g/some_things_just_never_change/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pezb3g</guid>
>       <pubDate>Tue, 31 Aug 2021 05:10:38 +0000</pubDate>
>     </item>
>     <item>
>       <title>Wait.... is this how it works?</title>
>       <link>https://i.redd.it/gp76waqiink71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf1goz/wait_is_this_how_it_works/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf1goz</guid>
>       <pubDate>Tue, 31 Aug 2021 08:07:13 +0000</pubDate>
>     </item>
>     <item>
>       <title>poor choice of words</title>
>       <link>https://i.redd.it/os8e8ots7pk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf69b0/poor_choice_of_words/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf69b0</guid>
>       <pubDate>Tue, 31 Aug 2021 13:50:39 +0000</pubDate>
>     </item>
>     <item>
>       <title>I see no difference</title>
>       <link>https://i.redd.it/pyaq5exfglk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pevm29/i_see_no_difference/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pevm29</guid>
>       <pubDate>Tue, 31 Aug 2021 01:11:57 +0000</pubDate>
>     </item>
>     <item>
>       <title>Monomie</title>
>       <link>https://v.redd.it/opha42os5pk71</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf62mj/monomie/">comments</a></p> <iframe src="https://v.redd.it/opha42os5pk71/DASH_1080.mp4?source=fallback" width="1080" height="1080"/> <img src="https://b.thumbs.redditmedia.com/of1xqhxOV27Of93vg3xKzMRYrivVqbqaLHJgIAT2JFE.jpg" class="webfeedsFeaturedVisual"/>]]></content:encoded>
>       <author>v.redd.it</author>
>       <guid>pf62mj</guid>
>       <pubDate>Tue, 31 Aug 2021 13:40:04 +0000</pubDate>
>     </item>
>     <item>
>       <title>Oh No, The Cursed Hairstyle</title>
>       <link>https://i.redd.it/6g6vnwro1lk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/peuf2k/oh_no_the_cursed_hairstyle/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>peuf2k</guid>
>       <pubDate>Tue, 31 Aug 2021 00:01:33 +0000</pubDate>
>     </item>
>     <item>
>       <title>Ahem... Tommorow is September.</title>
>       <link>https://i.redd.it/d17gvyqfmok71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf491n/ahem_tommorow_is_september/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf491n</guid>
>       <pubDate>Tue, 31 Aug 2021 11:51:13 +0000</pubDate>
>     </item>
>     <item>
>       <title>twitter</title>
>       <link>https://i.redd.it/nnw3d4h0wlk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pex276/twitter/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pex276</guid>
>       <pubDate>Tue, 31 Aug 2021 02:39:27 +0000</pubDate>
>     </item>
>     <item>
>       <title>no way</title>
>       <link>https://i.redd.it/r78geqc72pk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf5paa/no_way/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf5paa</guid>
>       <pubDate>Tue, 31 Aug 2021 13:19:17 +0000</pubDate>
>     </item>
>     <item>
>       <title>I don&#39;t know what to name it</title>
>       <link>https://i.redd.it/18mnx13p9pk71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf6gep/i_dont_know_what_to_name_it/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf6gep</guid>
>       <pubDate>Tue, 31 Aug 2021 14:01:17 +0000</pubDate>
>     </item>
>     <item>
>       <title>Yup!! No embarrassing stuff after this one!</title>
>       <link>https://i.redd.it/1s976n602mk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pexlth/yup_no_embarrassing_stuff_after_this_one/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pexlth</guid>
>       <pubDate>Tue, 31 Aug 2021 03:12:57 +0000</pubDate>
>     </item>
>     <item>
>       <title>Mosquito, thats what I see.</title>
>       <link>https://i.redd.it/rlh764gk4ok71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pf2w7x/mosquito_thats_what_i_see/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pf2w7x</guid>
>       <pubDate>Tue, 31 Aug 2021 10:10:51 +0000</pubDate>
>     </item>
>     <item>
>       <title>i wonder if it would melt...</title>
>       <link>https://i.redd.it/3d4r6l0zyik71.jpg</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pem1ck/i_wonder_if_it_would_melt/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pem1ck</guid>
>       <pubDate>Mon, 30 Aug 2021 16:50:32 +0000</pubDate>
>     </item>
>     <item>
>       <title>Since when did watching anime start making you cool</title>
>       <link>https://i.redd.it/otz69vnm4jk71.png</link>
>       <description></description>
>       <content:encoded><![CDATA[<p><a href="https://old.reddit.com/r/Animemes/comments/pemonj/since_when_did_watching_anime_start_making_you/">comments</a></p> ]]></content:encoded>
>       <author>i.redd.it</author>
>       <guid>pemonj</guid>
>       <pubDate>Mon, 30 Aug 2021 17:22:59 +0000</pubDate>
>     </item>
>   </channel>
> </rss>
`

**Comment by nikhilCad** [comment]
Multiple images (I think they are called galleries?) in one post are loading correctly. But single images don't load.

**Comment by trashhalo** [comment]
It appears to be working now. I added some extra logs that should let me know whats up. Let me know if it starts happening again and Ill dig in more

**Comment by nikhilCad** [comment]
Works here as well.  Was it a technical problem or are you being blocked by Reddit?

**Comment by trashhalo** [comment]
I'm not sure. I redployed and it started working again. Maybe the app got into a bad state? The new logs will help me better understand if it happens again.

**Comment by yutsuku** [comment]
Back to square one, unfortunately.
Attaching feed from old.reddit.com (I assume you get the feed from there judging by the source code), reddit.0qz.fun and a public instance of [teddit](https://codeberg.org/teddit/teddit/wiki#teddit-api) - teddit.net.

[0qz.feed.xml.log](https://github.com/trashhalo/reddit-rss/files/7117791/0qz.feed.xml.log)
[old-reddit.feed.xml.log](https://github.com/trashhalo/reddit-rss/files/7117795/old-reddit.feed.xml.log)
[teddit.feed.xml.log](https://github.com/trashhalo/reddit-rss/files/7117796/teddit.feed.xml.log)

**Comment by trashhalo** [comment]
Perfect! Now hopefully I have enough debugging information in the server to see what's really happening.

**Comment by trashhalo** [comment]
Fixed.

The work around for reddit blocking our feed was to stand up a small proxy server that hid that the HTTP requests were coming from our app. This tiny proxy server was falling over causing this error. This is why deploying fixed it because it brought the proxy back up. It appears reddit has removed their block requiring this proxy to begin with so I just removed it and it shouldnt happen again.

Thanks everyone for your patience.
